tests/poms/inventory.page.ts

```diff
@@ -19,15 +26,56 @@ export namespace InventoryService {
 		async close() {
 			await this.page.close();
 		}
+
+		async addProductToCart(addProductButtonTestId: string) {
+			const addProductButton = this.page.locator(
+				`[data-test="${addProductButtonTestId}"]`
+			);
+			await addProductButton.click();
+		}
+
+		async removeProductFromCart(removeProductButtonTestId: string) {
+			const removeProductButton = this.page.locator(
+				`[data-test="${removeProductButtonTestId}"]`
+			);
+			await removeProductButton.click();
+		}
+
+		async sortProductsBy(option: string) {
+			await this.page.selectOption('.product_sort_container', option);
+		}
+
+		async getProductNames() {
+			return this.productName.allInnerTexts();
+		}
+
+		async getProductPrices() {
+			const productPrices = await this.page.locator('.inventory_item_price');
+			const pricesText = await productPrices.allInnerTexts();
+			return pricesText.map((price) =>
+				parseFloat(price.replace('$', '').trim())
+			);
+		}
+
+		async clearCart() {
+			const response = await this.request.delete(`${baseURL}/api/cart`);
```

Comment:
doesn't seem to be correct, as the site itself doesn't use API but localstorage, hence remove it for now and implement clear local storage later